src/Entity/Jecoute/Survey.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace AppBundle\Entity\Jecoute;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\EntityIdentityTrait;
+use AppBundle\Entity\EntityTimestampableTrait;
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Ramsey\Uuid\Uuid;
+use Symfony\Component\Validator\Constraints as Assert;
+use JMS\Serializer\Annotation as JMS;
+
+/**
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\SurveyRepository")
+ * @ORM\Table(name="jecoute_survey")
+
+ * @Algolia\Index(autoIndex=false)
+ */
+class Survey
+{
+    use EntityIdentityTrait;
+    use EntityTimestampableTrait;
+
+    /**
+     * @ORM\Column
+     *
+     * @Assert\NotBlank
+     * @Assert\Length(max=255)
+     *
+     * @JMS\Groups({"survey_list"})
+     */
+    private $name;
+
+    /**
+     * The referent who created the survey.
+     *
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Adherent")
+     * @ORM\JoinColumn(onDelete="SET NULL")
+     */
+    private $referent;
```

Comment:
what about `author` instead of `referent`?

Reply:
I asked @JulHb , and we can use `creator`, what do you think ?